lambdas/db-migration/main.tf

```diff
@@ -13,7 +13,7 @@ data "aws_iam_policy_document" "lambda_assume_role_policy" {
 }
 
 resource "aws_iam_role" "db_migration" {
-  name                 = "${var.prefix}-db-migration"
+  name                 = "${var.prefix}-postgres-db-migration"
```

Comment:
These name changes are both more explicit *and* remove duplication contention with the CI/development stacks.